packages/hub/src/scripts/buildRecentModelRevision/worker.ts

```diff
@@ -0,0 +1,86 @@
+import { PrismaClient } from "@prisma/client";
+
+import { runSquiggle } from "@/graphql/queries/runSquiggle";
+import { ModelExport } from "@/lib/ExportsDropdown";
+
+const prisma = new PrismaClient();
+
+export async function runSquiggleCode(
+  currentRevisionId: string,
+  code: string,
+  seed: string
+): Promise<string | undefined> {
+  const outputR = await runSquiggle(code, seed);
+
+  if (outputR.ok) {
+    // I Imagine it would be nice to move this out of this worker file, but this would require exporting a lot more information. It seems wise to instead wait for the Serialization PR to go in and then refactor this.
```

Comment:
Since you already have IPC in place, it doesn't seem too hard to send `ModelExport[]` through it? You could also export a type from the worker and make IPC type-safe.

My main concern here is DB consistency: ideally, `ModelRevisionBuild` and `ModelExport` tables should be updated in a single transaction.

Reply:
Yea, we could now, as ModelExport isn't too complicated. Good point about the transactions.